### PR TITLE
(prod) DA-3536: updates in org service to support latest auth0 changes in da-libraries

### PR DESCRIPTION
**Pull request by ajinkyan83**

### CODE REVIEW

**Comment by ajinkyan83** [comment]
@lukaszgryglicki I will share the secrets over slack

**Review by enkhalifapro — APPROVED** [review]

File: main.go [hunk 1]
```diff
@@ -149,6 +153,8 @@ func initOrg() *orgservice.Org {
 		os.Getenv("AUTH0_CLIENT_SECRET"),
 		os.Getenv("AUTH0_AUDIENCE"),
 		os.Getenv("AUTH0_TOKEN_ENDPOINT"),
+		"",
```
Comment (lukaszgryglicki) [inline main.go:156]: If you added this new SLACK_WEBHOOK_URL, why didn't you pass it here?

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm - one comment, but I'm not sure if that comment is revelant or not.

File: main.go (hunk 1 above)
Comment (ajinkyan83) [reply main.go:156]: There are 2 new vars - one is the secret and one is slack url which is below this line..
As per discussion with @enkhalifapro, secret can be kept empty.

Comment (lukaszgryglicki) [reply main.go:156]: Even more, it mt be removed because it was duplicated param.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**